libbeat/filter/filter.go

```diff
@@ -65,6 +65,11 @@ func (filters *FilterList) Get(index int) FilterRule {
 // Applies a sequence of filtering rules and returns the filtered event
 func (filters *FilterList) Filter(event common.MapStr) common.MapStr {
 
+	// Check if filters are set, just return event if not
+	if filters == nil {
```

Comment:
@andrewkroh I changed it to

```
	// Check if filters are set, just return event if not
	if len(filters.filters) == 0 {
		return event
	}
```
Like this no cloning happens in case there are no filters.